### PR TITLE
feat(RELEASE-1060): fix linting issues in collect-data task

### PR DESCRIPTION
**Pull request by jinqi7**

### CODE REVIEW

File: .github/workflows/lint.yaml
Comment (mmalina) [inline .github/workflows/lint.yaml]: Opening a thread here so it's not accidentally merged with it.

File: tasks/collect-data/collect-data.yaml
Comment (mmalina) [inline tasks/collect-data/collect-data.yaml]: Rather than this, I would suggest using `bash` instead of `sh` above.

File: tasks/collect-data/collect-data.yaml
Comment (mmalina) [inline tasks/collect-data/collect-data.yaml]: Why these changes from echo to printf? If it's this:
```
^-- [SC3037](https://www.shellcheck.net/wiki/SC3037) (warning): In POSIX sh, echo flags are undefined.
```
Then again, using `bash` instead of `sh` should help.

File: tasks/collect-data/collect-data.yaml
Comment (mmalina) [inline tasks/collect-data/collect-data.yaml:162]: I suspect that once again, this is because `<<<` is not supported in POSIX `sh`. Use `bash` instead.

File: tasks/collect-data/collect-data.yaml
Comment (jinqi7) [reply tasks/collect-data/collect-data.yaml]: Thanks. I'll change to use `bash` .

File: tasks/collect-data/collect-data.yaml
Comment (jinqi7) [reply tasks/collect-data/collect-data.yaml]: Thanks.

File: tasks/collect-data/collect-data.yaml
Comment (mmalina) [inline tasks/collect-data/collect-data.yaml]: This is really more of a bugfix, so I would suggest 4.5.1 here.

**Review by mmalina — COMMENTED** [review]
Other than the minor nit about version, it looks good to me. (But the lint.yaml change needs to be reverted before I approve.)

**Review by johnbieren — COMMENTED** [review]
LGTM pending Martin's comments (both the version and github workflow change being removed)

File: tasks/collect-data/collect-data.yaml
Comment (jinqi7) [reply tasks/collect-data/collect-data.yaml]: Updated, thanks

File: .github/workflows/lint.yaml
Comment (jinqi7) [reply .github/workflows/lint.yaml]: Restored

**Review by mmalina — APPROVED** [review]

**Review by johnbieren — APPROVED** [review]

**Pull request merged**
